airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -260,7 +264,7 @@ def read_records(self, stream_slice: Optional[Mapping[str, Any]] = None, **kwarg
 
 
 class SourceStripe(AbstractSource):
-    def check_connection(self, logger, config) -> Tuple[bool, any]:
+    def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) -> Tuple[bool, any]:
```

Comment:
any -> Any

Reply:
@yevhenii-ldv can you make this change as part of your changes?